### ISSUE TITLE
Simplepush or Apprise notification agent

### ISSUE THREAD

**Issue by aniqueta** [JonnyWong16/plexpy#1595]
### Is your feature request related to a problem?

I believe there are no options yet for end-to-end encrypted (E2EE) notifications that work across platforms (iOS, Android, etc.)

### What is your feature request?

Could we implement notification agents for ideally [Simplepush](https://simplepush.io/features) which provides E2EE notifications across platforms? Alternative, implement support for [Apprise](https://github.com/caronc/apprise), which is middleware that can send notifications through a long list of services, including Simplepush. 

### Are there any workarounds?

One could conceivable develop a shell script to trigger Apprise, but it would be far less user friendly and customizable (like what Tautulli has for exiting notification agents). 

### Additional Context

@tymm

**Comment by Arcanemagus** [comment]
Note that the [Tautulli Remote](https://github.com/Tautulli/Tautulli-Remote) app is E2EE _if_ you have  the Python `pycryptodomex` module installed on the Tautulli server system (our Docker container / Windows .exe / macOS .pkg / snap package include this).

**Comment by aniqueta** [comment]
@Arcanemagus Thanks for the speedy reply! Did not realize that. Would this work on the macOS version of the server (assuming the right Python module is available)? Thanks.

**Comment by Arcanemagus** [comment]
Yep, it's part of the packages included in all of the pre-made bundles we provide:
https://github.com/Tautulli/Tautulli/blob/a944f5ae06d4f0bd67cd65b83e7389325ca313b6/package/requirements-package.txt#L5

**Comment by aniqueta** [comment]
Should this be closed since my/the E2EE use case was addressed? Or is there still value in having an issue open for Simplepush and Apprise? Thanks.

**Comment by JonnyWong16** [comment]
Simplepush could happen. Apprise probably won't because the notifications are quite limited and can't provide the customization that Tautulli allows.

**Comment by tivvit** [comment]
Hi, I am not an expert, and a lot may have changed in the last few years. However, Apprise should support a lot and can route to almost anything, so I think that would be a great solution.
